### PR TITLE
#4 ATValidator + tests

### PR DESCRIPTION
**Pull request by harfmuf**

### CODE REVIEW

File: src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    public Pattern getPattern(){
+        return PATTERN;
+    }
+
+    public boolean computeControlSum(String tin) {
+        int[] weights = {1, 2, 1, 2, 1, 2, 1, 2};
+        try {
+            int sum = 0;
+            for (int i = 0; i < weights.length; i++) {
+                int n = Integer.parseInt(tin.substring(i, i + 1)) * weights[i];
```
Comment (DcortezMeleth) [inline src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java]: Mechanism of multiplication by weights is quit popular there. Maybe we can leverage that to some kind of util or method in a parent class?

File: src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.onwelo.tinvalidator.validator;
+
+import java.util.regex.Pattern;
+
+public class ATValidator implements Validator {
+
+    private static final Pattern PATTERN = Pattern.compile("[0-9]{9}");
```
Comment (DcortezMeleth) [inline src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java]: Rozumiem że znaki `-` i `\` są wycinane gdzieś wyżej?

File: src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java (hunk 1 above)
Comment (harfmuf) [reply src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java]: despite mechanism is similar, there are many details of difference and probably the only common code is exactly two above lines, but `for` body often differs. I think we should keep it separate.

File: src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java (hunk 2 above)
Comment (harfmuf) [reply src/main/java/com/onwelo/tinvalidator/validator/ATValidator.java]: in TinValidator class all not alphanumeric characters are removed

**Pull request merged**